src/components/movie-list-page/movie-list-page.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { lazy, Suspense, useEffect, useState } from 'react';
+import MovieDetails from '../movie-details/movie-details';
+import Dialog from '../dialog/dialog';
+import FormMovie from '../forms/form-movie/form-movie';
+import SearchForm from '../search-form/search-form';
+import GenreSelect from '../genre-select/genre-select';
+import SortControl from '../sort-control/sort-control';
+import DeleteMovie from '../forms/delete-movie/delete-movie';
+import MovieTile from '../movie-tile/movie-tile';
+import './movie-list-page.scss';
+import { getMovies, searchMovies, sortMovies } from '../../services/services';
+
+const MovieContextEnum = {
+  edit: 'edit',
+  delete: 'delete'
+};
+
+const MovieListPage = () => {
+  // const [searchQuery, setSearchQuery] = useState('');
+  // const [sortCriterion, setSortCriterion] = useState('');
+  const [activeGenre, setActiveGenre] = useState('');
```

Comment:
Changing genres doesn't work. You need to make requests for data after changing it